z2/src/deployer.rs

```diff
@@ -144,6 +153,10 @@ async fn execute_install_or_upgrade(
     Ok(())
 }
 
+async fn post_install(node: ChainNode) -> Result<()> {
```

Comment:
is there any reason not to call `node.post_install().await` directly in the change above?